webapp/channels/src/plugins/useGetPluginsActivationState.ts

```diff
@@ -12,14 +12,12 @@ export const useGetPluginsActivationState = () => {
     const pluginsList = useSelector((state: GlobalState) => state.plugins.plugins);
     const pluginProducts = useProducts();
 
-    let boardsProductEnabled = false;
     let playbooksProductEnabled = false;
     if (pluginProducts) {
-        boardsProductEnabled = pluginProducts.some((product) => (product.pluginId === suitePluginIds.focalboard) || (product.pluginId === suitePluginIds.boards));
```

Comment:
I think we also want to keep this line, but with only the check for `suitePluginIds.focalboard`

Reply:
I would've thought that it was different, but because it checked for both IDs, it would've been true for both plugin and product mode. `playbooksProductEnabled` also would've been true for both plugin and product mode as well, so I don't know what the difference between those two were either